client/src/endToEndManualTest/java/io/obswebsocket/community/client/test/ObsRemoteE2eObservationIT.java

```diff
@@ -1,223 +1,248 @@
+  }
+
+//  TODO: How to do this?
 //  @Test
 //  void playPauseVlcMedia() {
+//    Number vlcId = remote.getSceneItemId(SCENE1, SOURCE_VLC_MEDIA, 0, 1000).getSceneItemId();
 //    obsShould("Play video 1, by showing it (Note, VLC must be installed!)");
-//    remote.setSourceVisibility(null, SOURCE_VLC_MEDIA, true, loggingCallback);
+//    remote.setSceneItemEnabled(SCENE1, vlcId, true, loggingCallback());
 //
 //    obsShould("Pause video 1");
-//    remote.pauseMedia(SOURCE_VLC_MEDIA, loggingCallback);
+//    remote.pauseMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //
 //    // BUG: Toggle Play/Pause does not work in Palakis OBS plugin!
 //    // see https://github.com/obsproject/obs-websocket/issues/725
 //    // I've noted we can also replicate the problem
 //    obsShould("Toggle Play Video 1");
-//    remote.toggleMedia(SOURCE_VLC_MEDIA, loggingCallback);
+//    remote.toggleMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //    obsShould("Toggle Pause Video 1");
-//    remote.toggleMedia(SOURCE_VLC_MEDIA, loggingCallback);
+//    remote.toggleMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //
 //    obsShould("Switch to video 2");
-//    remote.nextMedia(SOURCE_VLC_MEDIA, loggingCallback);
+//    remote.nextMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //
 //    obsShould("Restart, back at video 1 (should auto play)");
-//    remote.restartMedia(SOURCE_VLC_MEDIA, loggingCallback);
+//    remote.restartMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //
 //    obsShould("Stop video 1 (going back to beginning)");
-//    remote.stopMedia(SOURCE_VLC_MEDIA, loggingCallback);
-//
-//  }
-//
-//  @Test
-//  void triggerHotkey() {
-//    obsShould("Show the red square via hotkey name");
-//    remote.triggerHotkeyByName("libobs.show_scene_item.red_square", loggingCallback);
+//    remote.stopMedia(SOURCE_VLC_MEDIA, loggingCallback());
 //  }
-//
-//  @Test
-//  void refreshBrowserSource() {
-//
-//    obsShould("Show the browser source", 1);
-//    remote.setSourceVisibility(null, SOURCE_BROWSER, true, loggingCallback);
-//
+
+  @Test
+  void triggerHotkey() {
+    Number squareId = remote.getSceneItemId(SCENE1, SOURCE_RED_SQUARE, 0, 1000).getSceneItemId();
+
+    obsShould("Show the red square via hotkey name");
+    remote.triggerHotkeyByName("libobs.show_scene_item.red_square", loggingCallback());
+
+    obsShould("Hide the square again");
+    remote.setSceneItemEnabled(SCENE1, squareId, false, loggingCallback());
+  }
+
+  @Test
+  void refreshBrowserSource() {
+    Number browserId = remote.getSceneItemId(SCENE1, SOURCE_BROWSER, 0, 1000).getSceneItemId();
+
+    obsShould("Show the browser source", 1);
+    remote.setSceneItemEnabled(SCENE1, browserId, true, loggingCallback());
+
+    // TODO: How to do this?
 //    obsShould("Refresh the browser source (new random color and number)");
-//    remote.refreshBrowserSource(SOURCE_BROWSER, loggingCallback);
+//    remote.refreshBrowserSource(SOURCE_BROWSER, loggingCallback());
```

Comment:
.pressInputPropertiesButton(SOURCE_BROWSER, "refreshnocache", ...)

Reply:
Works and fixed